aws/modules/cloudwatchslack/module.ftl

```diff
@@ -47,6 +47,16 @@
             "Description" : "A sha1 hash of the lambda image to ensure its the right one",
             "Default" : "8f194db4f6ed2b826387112df144f188451ba6db"
         },
+        {
+            "Names" : "lambdaRuntime",
+            "Type" : STRING_TYPE,
+            "Description" : "runtime engine for lambda",
+            "Values" : [
+                "python3.6",
+                "python3.9"
+            ],
+            "Default" : "python3.6"
+        },
         {
```

Comment:
We should just set the lambda RunTime to Python3.9. The code doesn't have any new features in it and python3.9 is backwards compatible with python3.6 

There is no impact in changing runtime between these two and no need for choice.

Reply:
Correct, but earlier versions of hamlet did not support above 3.8. Do all "in use" versions of hamlet support 3.9 (I'm particularly thinking out HA)